homeassistant/components/teletask/light.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for Teletask/IP lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.teletask/
+"""
+
+import voluptuous as vol
+
+from homeassistant.components.teletask import (DATA_TELETASK)
+from homeassistant.components.light import (PLATFORM_SCHEMA,
+                                            ATTR_BRIGHTNESS,
+                                            SUPPORT_BRIGHTNESS,
+                                            Light)
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+
+CONF_ADDRESS = 'address'
+CONF_DOIP_COMP = 'doip_component'
+CONF_BRIGHTNESS_ADDRESS = 'brightness_address'
+
+DEFAULT_NAME = 'Teletask Light'
+DEPENDENCIES = ['teletask']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ADDRESS): cv.string,
+    vol.Optional(CONF_BRIGHTNESS_ADDRESS, default=""): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_DOIP_COMP, default="relay"): cv.string
+})
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info):
+    """Set up lights for Teletask platform."""
+    await async_add_entities_config(hass, config, async_add_entities)
+
+
+@callback
+async def async_add_entities_config(hass, config, async_add_entities):
+    """Set up light for Teletask platform configured within platform."""
+    import teletask
+    light = teletask.devices.Light(
+        teletask=hass.data[DATA_TELETASK].teletask,
+        name=config.get(CONF_NAME),
+        group_address_switch=config.get(CONF_ADDRESS),
```

Comment:
Use `dict[key]` for required config keys and keys with default config schema values.